app/lib/features/chat_ng/widgets/events/text_message_event.dart

```diff
@@ -17,16 +17,20 @@ class TextMessageEvent extends StatelessWidget {
   final String roomId;
   final MsgContent content;
   final TextMessageType _type;
-  final bool _isUser; // Only needed for emoji messages to determine style
+  final bool _isMe; // Only needed for emoji messages to determine style
+  final String? displayName;
+  final Widget? repliedTo;
```

Comment:
As we want the order to be `DisplayName?`, `RepliedTo?`, `Message`, the replies have to rendered from within here rather than in the ChatBubble (which IMHO also makes more sense structurally)